docs/UserGuide.adoc

```diff
@@ -79,14 +79,17 @@ Format: `find KEYWORD [MORE_KEYWORDS]`
 
 [NOTE]
 ====
-The search is case sensitive, the order of the keywords does not matter, only the name is searched,
+The search is not case sensitive, the order of the keywords does not matter, only the name is searched,
```

Comment:
I think this is a good idea, makes it easier to search for people without bothering with upper or lower case

Reply:
Hi, Jia Hao.

Maybe you want to use inline comments for more specific things. For overall feedback, leave it in the review summary.